xs2a-adapter-rest-impl/src/main/resources/static/main.js

```diff
@@ -73,28 +73,28 @@ function search() {
 
     //  no need for now
     // if (data[3].value !== "")
-    //     url += "scaApproaches=" + data[3].value.toLowerCase();
+    //     url += "approach=" + data[3].value.toLowerCase();
```

Comment:
Should we keep this unused code in the repo?

Reply:
Mark may need it in the future (or may not)

It's easy to reproduce so can be safely deleted